payments/mobile-pay/index.md

```diff
@@ -16,214 +16,63 @@ sidebar:
-site when the purchase is fulfilled." %}
-
-When dealing with credit card payments, 3-D Secure authentication of the
-cardholder is an essential topic.
-There are two alternative outcome of a credit card payment:
-
-* 3-D Secure enabled - by default, 3-D Secure should be enabled, and Swedbank
-  Pay will check if the card is enrolled with 3-D Secure.
-  This depends on the issuer of the card.
-  If the card is not enrolled with 3-D Secure, no authentication
-  of the cardholder is done.
-* Card supports 3-D Secure - if the card is enrolled with 3-D Secure,
-  Swedbank Pay will redirect the cardholder to the autentication mechanism
-  that is decided by the issuing bank.
-  Normally this will be done using BankID or Mobile BankID.
-
-```mermaid
-sequenceDiagram
-  Consumer->>MerchantOrderSystem: consumer starts purchase
-  Activate MerchantOrderSystem
-  MerchantOrderSystem->>Merchant: start purchase process
-  Activate Merchant
-  Merchant->>PayEx: POST [payment] (operation=PURCHASE)
-  note left of Merchant: First API request
-  Activate PayEx
-  PayEx-->>Merchant: payment resource with payment URL
-  Deactivate PayEx
-  Merchant-->>MerchantOrderSystem: Payment URL sent to order system
-  Deactivate Merchant
-  MerchantOrderSystem-->>Consumer: Distribute Payment URL through e-mail/SMS
-  Deactivate MerchantOrderSystem
-
-  note left of Consumer: Payment Link in e-mail/SMS
-  Consumer->>PayEx: Open link and enter payment information
-  Activate PayEx
-
-  opt Card supports 3-D Secure
-  PayEx-->>Consumer: redirect to IssuingBank
-  Deactivate PayEx
-  Consumer->>IssuingBank: 3-D Secure authentication process
-  Consumer->>PayEx: access authentication page
-  Activate PayEx
-  end
-
-  PayEx-->>Consumer: redirect to merchant site
-  note left of PayEx: redirect back to merchant
-  Deactivate PayEx
-
-  Consumer->>Merchant: access merchant page
-  Activate Merchant
-  Merchant->>PayEx: GET [payment]
-  note left of Merchant: Second API request
-  Activate PayEx
-  PayEx-->>Merchant: payment resource
-  Deactivate PayEx
-  Merchant-->>Consumer: display purchase result
-  Deactivate Merchant
-```
+After the payment is created, the consumer is redirected to Mobile Pay's
+own payment page where the phone number is entered, and a push message is sent
+to the phone.
+
+![screenshot of the Mobile Pay number input page][mobilepay-number-input]{:height="600px" width="425px"}
+
+A countdown is shown in the browser and the payment request will appear in the
+app, waiting to be confirmed by the consumer. If you are using the redirect
+option, the consumer will be redirected back to the merchant's site.
+
+## Good To Know
+
+### Payment Type
+
+Mobile Pay is one of the instruments using two-phase payments. The
+`authorization` is done when the consumer successfully confirms in the app, and
+the `abort`, `cancel`, `capture` or `reversal` is done by the merchant at a
+later time. Read more about the [different operations][after-payment] and the
```

Comment:
Perhaps the `different operations` link should go directly to the `#operations` header and not just to the top of the `after payment` page?